frontend/src/container/ForgotPassword.jsx

```diff
@@ -80,10 +80,11 @@ export class ForgotPassword extends React.Component {
 
   handleClickSubmit = async () => {
     const { props, state } = this
+    const email = state.backupLogin.value.trim()
```

Comment:
Same here, I would leave the variable name as "login" or "backupLogin" because in this case it can be an email or a username.

Reply:
The thing is, this variable is only used as an email, on line 87, if it's an email then I use this variable, overwise i use the state.backupLogin.value